### PR TITLE
Update Sass to 3.6.4

### PR DESCRIPTION
**Pull request by MasterEnoc**
- [x] Updated the versions and releases in the scraper file
- [x] Ensured the license is up-to-date and that the documentation's entry in the array in `about_tmpl.coffee` matches it's data in `self.attribution`
- [x] Ensured the icons and the `SOURCE` file in <code>public/icons/*your_scraper_name*/</code> are up-to-date if the documentation has a custom icon
- [x] Ensured `self.links` contains up-to-date urls if `self.links` is defined
- [x] Tested the changes locally to ensure:
  - The scraper still works without errors
  - The scraped documentation still looks consistent with the rest of DevDocs
  - The categorization of entries is still good

### CODE REVIEW

File: lib/docs/filters/sass/entries.rb [hunk 1]
```diff
@@ -1,89 +1,53 @@
...
       def get_type
-        'Functions'
+        return 'Syntax' if slug.include?('syntax')
+        return 'Style rules' if slug.include?('style')
+        return 'At-Rules' if slug.include?('at')
+        return 'Values' if slug.include?('values')
+        return 'Operators' if slug.include?('operators')
+        return 'Command line' if slug.include?('cli')
+        return 'Modules' if slug.include?('modules')
+        'Misc'
```
Comment (scherepn) [inline lib/docs/filters/sass/entries.rb:17]: This is only a code style thing, but a cleaner way to do this is with this code:
```ruby
case slug
when /syntax/
  'Syntax'
when /style/
  'Style'
...
else
  'Misc'
end
```

Comment (MasterEnoc) [reply lib/docs/filters/sass/entries.rb:17]: Thank you!, any suggestion to improve the quality of the code is welcome

File: lib/docs/scrapers/sass.rb [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
-    options[:only] = %w(Sass/Script/Functions.html)
     options[:root_title] = false
     options[:title] = 'Sass Functions'
 
-    options[:container] = ->(filter) do
-      filter.root_page? ? '#filecontents' : '#instance_method_details'
-    end
+    options[:skip_patterns] = [/breaking-changes/]
+
+    # options[:container] = '#main-content'
```
Comment (scherepn) [inline lib/docs/scrapers/sass.rb:19]: Was this line accidentally left in?

**Review by simon04 — APPROVED** [review]
Thank you!

**Pull request merged**